cypress/integration/trakt/comments.spec.js

```diff
@@ -33,7 +33,9 @@ describe('04: Updates, reply and likes on comments.', () => {
 			expect(response.status).to.be.eq(201)
 			expect(response.body).to.have.property('id')
 
-			cy.deleteComment(response.body.id)
+			cy.deleteComment(response.body.id).then(resp => {

```

Comment:
There is no need to delete this comment once you created it. In case you are going to test later another scenario related to comments, you can use another movie to comment or if you will check the number of comments, you can assert if the list of comments length is greater or equal to 1.

Remember: the best practice indicates that it is better to reset the state of the system before each test and not after each test.